doc/ExpertOptions.md

```diff
@@ -1549,3 +1550,52 @@ the existing version can be used. The checksum covers the WDL source
 code, the DNAnexus runtime specification, and any other attributes. There
 are two exceptions: the project name, and the folder. This allows
 moving WDL workflows in the folder hierarchy without recompilation.
+
+# Publishing global workflows
+
+- _TODO_ Basic usage example, compile
+
+- _TODO_ Basic usage example, build global
+
+Create global workflows with `dx build --globalworkflow --from workflow-yyyy --version X.Y.Z --bill-to <org or user>`.
+
+- _TODO_ What will be cloned with the workflow
+
+- _TODO_ What will not be cloned (see Limitations for more details)
+
+## Recommendations
+
+- _TODO_ Developers should not store any credentials and passwords in the source code of the gwf as authorized users can download any applets referenced by the gwf
+
+- _TODO_ Developers should use simple types for input/output interfaces for global workflows for better platform experience.
+
+- _TODO_ Specify runtime resources using memory, disk, cpu requirements (not hard-coded instance types) for better compatibility across runtime projects.
+
+- _TODO_ Compile with `-instanceTypeSelection dynamic` for better compatibility across runtime projects, but expect longer runtime.
+
+- _TODO_ The developer of a global WDL-based workflow should include a pointer to WDL git repo commit that the workflow is based on for informational purposes by updating the description of the global workflow if needed.
+
+## Limitations
+
+- _TODO_ What will not be cloned with the workflow
+
+Review the dependencies to make sure they are accessible to authorized users of the workflow
+
+- _TODO_ How to view the Markdown description until UI page works
+
+I think the biggest UI issue is lack of an easy way to look at pretty markdown encoded in the description of the applets and workflows in UI or CLI.
+I had to resort to this. Need to document this.
+
+```
+dx describe globalworkflow-<name>/<version> --json | jq -rc '.description | tostring'
+```
+
+- _TODO_ Is there a correct way to reference an id of an applet that is part of a global workflow (exists in the global workflow's resource container) that will not stress the API server with a global search? Do we have to instruct the user here how to find the project id of the resource container first?
```

Comment:
Documentation question. Is there a correct way to reference an id of an applet that is part of a global workflow (exists in the global workflow's resource container) that will not stress the API server with a global search? Do we have to instruct the user here how to find the project id of the resource container first? @commandlinegirl @emiloslavsky @asimenos

Reply:
Actually, regarding dx describe applet-xxxx, I think I will only instruct the user to dx describe globalworkflow-<name>/<version> since the relevant part of the description for all tasks and subworkflows is aggregated there. They should be able to find the info they need without describing applets individually.